frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -2692,4 +2694,18 @@ private void createModRequest(ResourceRequest request, ResourceResponse response
             response.setProperty(ResourceResponse.HTTP_STATUS_CODE, "500");
         }
     }
+
+    private Set<String> getProjectGroups(List<Project> projects) {
+        Set<String> businessUnit = new HashSet<String>();
+        try {
+            if(!CommonUtils.isNullOrEmptyCollection(projects)) {
+                for(Project project : projects) {
+                    businessUnit.add(project.getBusinessUnit());
+                }
+            }
+        } catch (Exception ex) {
```

Comment:
Is it required to catch Exception because there is no checked Exception thrown.
Also this process is Primary process, and not a secondary process like `logging`, `changelogs` etc.

Reply:
Removed the try catch block.